source/common/protobuf/utility.cc

```diff
@@ -589,10 +589,14 @@ void MessageUtil::checkForUnexpectedFields(const Protobuf::Message& message,
 std::string MessageUtil::getYamlStringFromMessage(const Protobuf::Message& message,
                                                   const bool block_print,
                                                   const bool always_print_primitive_fields) {
-  std::string json = getJsonStringFromMessage(message, false, always_print_primitive_fields);
+
+  auto json_or_error = getJsonStringFromMessage(message, false, always_print_primitive_fields);
+  if (!json_or_error.ok()) {
```

Comment:
Can we just use the `OrDie` variant here? Is there anyone who can reasonably catch this exception?

Reply:
No idea, I just figured that since this is already throwing exceptions, we can do that instead of crash.